crates/compiler/builtins/bitcode/src/main.zig

```diff
@@ -145,6 +145,7 @@ comptime {
     exportStrFn(str.strTrimRight, "trim_right");
     exportStrFn(str.strCloneTo, "clone_to");
     exportStrFn(str.withCapacity, "with_capacity");
+    exportStrFn(str.strGraphemes, "str_graphemes");
```

Comment:
Could you rename this to remove the redundant prefix `str_`?
We already prefix it with `str.` inside `exportStrFn` so this is not necessary and none of the other exports do it.

Reply:
Otherwise I don't see any issues, looks good! 👍